client/my-sites/plan-features/index.jsx

```diff
@@ -288,6 +307,42 @@ class PlanFeatures extends Component {
 		} );
 	}
 
+	renderPopover() {
+		return (
+			<Popover
+				id="popover__plan-features"
+				isVisible={ this.state.showPopover }
+				context={ this.state.popoverReference }
+				position="right"
+				onClose={ this.closePopover }
+				className={ classNames(
+						'info-popover__tooltip',
+						'popover__plan-features'
+					) }
+				>
+					{ this.state.popoverDescripcion }
+			</Popover>
+		);
+	}
+
+	showPopover( el, popoverDescripcion ) {
+		const shouldHide = this.state.popoverReference === el &&
```

Comment:
We could store the reference into `<Item />` but we would be propagating the same reference to all of them and my idea here is simplify trying to avoid unnecessary code. I understand this scenario like a lot of targets tied to only one Popover so where we have only one reference, only one visibility status, and so on. These references should be reside in the same scope where the Popover instance is created.

**before**
<img src="https://cloud.githubusercontent.com/assets/77539/17809174/23ac075e-65eb-11e6-8a8f-e6e850e0ef20.png" width="400px" />